src/Apps/Artwork/Components/PricingContext.tsx

```diff
@@ -19,6 +19,10 @@ function PricingContext({ artwork }: PricingContextProps) {
     return null
   }
 
+  const priceCents = artwork.priceCents.max
```

Comment:
Do we have to worry about `artwork.priceCents.min` being `null` here as well?

Reply:
Nope :) If there's no 'min' price then there's no price whatsoever and we don't return histogram data from metaphysics.